File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/commands/CommitSeatReservation.cs
using System;
using ENode.Commanding;

namespace ConferenceManagement.Commands
{
    public class CommitSeatReservation : Command<Guid>
    {
        public Guid ReservationId { get; set; }

        public CommitSeatReservation() { }
        public CommitSeatReservation(Guid conferenceId, Guid reservationId) : base(conferenceId)
        {
            ReservationId = reservationId;
        }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/commands/CommitSeatReservation.java
package com.microsoft.conference.common.management.commands;

import org.enodeframework.commanding.Command;

public class CommitSeatReservation extends Command<String> {
    public String reservationId;

    public CommitSeatReservation() {
    }

    public CommitSeatReservation(String conferenceId, String reservationId) {
        super(conferenceId);
        this.reservationId = reservationId;
    }
}
